Packs/XSOAR_EDL_Checker/Integrations/XSOAREDLChecker/README.md

```diff
@@ -1,5 +1,5 @@
-Checks an XSOAR EDL to make sure it's returning a valid response.
-This integration was integrated and tested with version 6.8+ of Cortex XSOAR.
+Checks an XSOAR hosted EDL to make sure it's returning a valid response.   Supports PAN-OS (text), CSV, or JSON EDLs.
+This integration was integrated and tested with version xx of XSOAR EDL Checker.
```

Comment:
```suggestion
This integration was integrated and tested with version ????? of XSOAR EDL Checker.
```
Please replace with the correct version.

Reply:
As this integration works with Cortex XSOAR and not an external system, updated to say which versions of XSOAR it was tested with (6.12 and 8.4) as well as the version of the Generic Export Indicators Service that provides the EDLs.